spec/unit/property/list_spec.rb

```diff
@@ -32,11 +32,11 @@
     end
 
     it "should return the passed in array values joined with the delimiter from is_to_s" do
-      expect(@property.is_to_s(["foo","bar"])).to eq("foo,bar")
+      expect(@property.is_to_s(['foo','bar'])).to eq("['foo', 'bar']")
     end
 
-    it "should be able to correctly convert ':absent' to a string" do
-      expect(@property.is_to_s(:absent)).to eq("absent")
```

Comment:
Absent (present, latest, etc) bare words are special, I think those should remain unquoted, e.g. we write:

```puppet
file { '/tmp/a':
  ensure => absent
}
```

but not `ensure => 'absent'`